### PR TITLE
basic delegations button styling and visibility

### PR DESCRIPTION
**Pull request by jinjagit**
Addresses some of #9 

Simply adding HTML `class="button"` and HTML `style="background:orange"` for some more consistent styling (relative to Decidim styling). Button sizes are not yet consistent with div layout, nor responsive to div size.  
  
Also adding conditional to `_vote_button.html.erb` to hide create delegation button when a delegation exists.

### CODE REVIEW

**Comment by jinjagit** [comment]
I am currently making a screenshot to put here.

File: app/views/decidim/proposals/proposals/_vote_button.html.erb [hunk 1]
```diff
@@ -1,25 +1,27 @@
 <% if session[:delegated_to] %>
-Support delegated to <%= session[:delegated_to] %>
-<form action="/liquidvoting/delegations" method="POST">
-  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
-  <input type="hidden" name="delegator_email" value="<%= current_user.email %>">
-  <input type="hidden" name="proposal_url" value="<%= proposal_url(proposal) %>">
-  <input name="_method" type="hidden" value="delete">
-  <input type='submit' value='Delete Delegation'>
-</form>
+  Support delegated to <%= session[:delegated_to] %>
```
Comment (jinjagit) [inline app/views/decidim/proposals/proposals/_vote_button.html.erb:2]: Oops, sorry. Dididn't realize my cut /copy /paste would result in all these lines appearing to have changed. The relevant changes are highlighted, below.

File: app/views/decidim/proposals/proposals/_vote_button.html.erb [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
-  <input type="hidden" name="delegator_email" value="<%= current_user.email %>">
-  <input type="hidden" name="proposal_url" value="<%= proposal_url(proposal) %>">
-  <select name="delegate_email">
-  <% Decidim::User.where(admin: false).order(:name).each do |user| %>
-    <option value="<%= user.email %>"><%= user.name -%></option>
-  <% end %>
-  </select>
-  <input type='submit' value='Choose a Delegate'>
-</form>
+<% if !session[:delegated_to] %>
```
Comment (jinjagit) [inline app/views/decidim/proposals/proposals/_vote_button.html.erb:12]: inserted this line

File: app/views/decidim/proposals/proposals/_vote_button.html.erb [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
+    <input type="hidden" name="delegator_email" value="<%= current_user.email %>">
+    <input type="hidden" name="proposal_url" value="<%= proposal_url(proposal) %>">
+    <select name="delegate_email">
+    <% Decidim::User.where(admin: false).order(:name).each do |user| %>
+      <option value="<%= user.email %>"><%= user.name -%></option>
+    <% end %>
+    </select>
+    <input type='submit' value='Choose a Delegate' class="button" style="background:orange;">
+  </form>
+<% end -%>
```
Comment (jinjagit) [inline app/views/decidim/proposals/proposals/_vote_button.html.erb:24]: Inserted this line.

File: app/views/decidim/proposals/proposals/_vote_button.html.erb [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
+  <form action="/liquidvoting/delegations" method="POST">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+    <input type="hidden" name="delegator_email" value="<%= current_user.email %>">
+    <input type="hidden" name="proposal_url" value="<%= proposal_url(proposal) %>">
+    <select name="delegate_email">
+    <% Decidim::User.where(admin: false).order(:name).each do |user| %>
+      <option value="<%= user.email %>"><%= user.name -%></option>
+    <% end %>
+    </select>
+    <input type='submit' value='Choose a Delegate' class="button" style="background:orange;">
```
Comment (jinjagit) [inline app/views/decidim/proposals/proposals/_vote_button.html.erb:22]: Edited this line.

**Review by oliverbarnes — APPROVED** [review]
LGTM. Great start, already feels more user friendly

**Pull request merged**